packages/zendesk-adapter/src/config_opt.ts

```diff
@@ -0,0 +1,59 @@
+/*
+*                      Copyright 2022 Salto Labs Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import { BuiltinTypes, ConfigOpt, ElemID, InstanceElement, ObjectType } from '@salto-io/adapter-api'
+import { createDefaultInstanceFromType, safeJsonStringify } from '@salto-io/adapter-utils'
+import { logger } from '@salto-io/logging'
+import { configType } from './config'
+import * as constants from './constants'
+
+const log = logger(module)
+
+const configOptElemId = new ElemID(constants.ZENDESK, 'configOptType')
+
+export const configOptObjectType = new ObjectType({
+  elemID: configOptElemId,
+  fields: {
+    enableGuide: { refType: BuiltinTypes.BOOLEAN },
+  },
+})
+
+const isConfigOptInstance = (instance: InstanceElement): boolean => {
```

Comment:
if this is a typeguard then let's make it a proper typeguard? 
```suggestion
const isConfigOptInstance = (instance: InstanceElement): instance is InstanceElement & { value: ConfigCustomizationType } => {
```